gradle/formatting.gradle

```diff
@@ -82,7 +82,7 @@ allprojects {
               '\\#java|\\#org.opensearch|\\#org.hamcrest|\\#'
           )
 
-          eclipse().configFile rootProject.file('buildSrc/formatterConfig.xml')
+          eclipse().withP2Mirrors(Map.of("https://download.eclipse.org/", "https://mirror.umd.edu/eclipse/")).configFile rootProject.file('buildSrc/formatterConfig.xml')
```

Comment:
Does it make sense to add any other mirrors?

https://download.eclipse.org/oomph/archive/mirror.php

Reply:
Lets keep an eye and add them back again @msfroh WDYT?